tests/test_cert_utils.py

```diff
@@ -5,8 +5,15 @@ def test_wildcard():
     assert to_wildcard_domain("localhost") == "localhost"
     assert to_wildcard_domain("cosmian.app") == "cosmian.app"
     assert to_wildcard_domain(".cosmian.app") == "*.cosmian.app"
-    assert to_wildcard_domain("9a5029d5-f769-4749-804e-50f6711bd509.cosmian.app") == "*.cosmian.app"
-    assert to_wildcard_domain(
-        "9a5029d5-f769-4749-804e-50f6711bd509.dev.cosmian.app") == "*.dev.cosmian.app"
-    assert to_wildcard_domain(
-        "9a5029d5-f769-4749-804e-50f6711bd509.stagging.cosmian.app") == "*.stagging.cosmian.app"
+    assert (
+        to_wildcard_domain("9a5029d5-f769-4749-804e-50f6711bd509.cosmian.app")
+        == "*.cosmian.app"
+    )
+    assert (
+        to_wildcard_domain("9a5029d5-f769-4749-804e-50f6711bd509.dev.cosmian.app")
+        == "*.dev.cosmian.app"
+    )
+    assert (
+        to_wildcard_domain("9a5029d5-f769-4749-804e-50f6711bd509.stagging.cosmian.app")
+        == "*.stagging.cosmian.app"
```

Comment:
`staging`

Reply:
Fixed in f438dd769553cbb6251098fce161bf95177ba429.